src/core/tests/Test_CoreMain.py

```diff
@@ -65,7 +65,7 @@ def test_operation_fail_for_non_autopatching_request(self):
         self.__check_telemetry_events(runtime)
 
         # check status file
-        with runtime.env_layer.file_system.open(runtime.execution_config.status_file_path, 'r') as file_handle:
```

Comment:
All the tests checking .status file should remain as is. 

The client/agent facing output is .status file and not .complete.status. Hence we need to check and assert things within .status file. 

Please revert the changes from status_file_path to complete_status_file_path in all tests